### DISCUSSION TITLE
When will Rustdesk 1.2.0 reach stable?

### DISCUSSION THREAD

**Discussion by Darin755** [rustdesk/rustdesk#3319] [Q&A]
I have been testing the beta release and it seems pretty stable. I love the new features and it seems like a big improvement.

When will it become stable? I know this is a hard question to answer but I think it would be good to have a rough timeline

**Comment by adamency** [comment]
Indeed, @rustdesk, the project is amazing in terms of feature, but it would be great to have communication on the advancement of the project. No new version has been released since may 2022 while development seems to have exploded since then, according to the Contributors graph, which is a bit counterintuitive. I also see that `nightly` is being updated regularly, but the release itself is marked at Oct 2022 so you have to literally go the nightly page to realize that is actually more up to date. Finally there is no changelog to be able to read about the new features, while many people seem to agree that the nightly is quite stable.

@Darin755 Btw how did you learn about the new features ? Do you just build and test the project ? How to know new commands in the CLI for instance ? There's no documentation except for installing and no changelog. It seems the only way to understand the program is to binge read issues and discussions currently, if you're not a rust dev.

A big and simple thing right now given the situation would be to at least indicate the relative stability of using `nightly` from one side to `1.1.9` to the other side and vice-versa, given that our non-tech peers will most likely be using the latter.

**Comment by Darin755** [comment]
> @Darin755 Btw how did you learn about the new features ? Do you just build and test the project ? How to know new commands in the CLI for instance ? There's no documentation except for installing and no changelog. It seems the only way to understand the program is to binge read issues and discussions currently, if you're not a rust dev.

I am just comparing the nightly version to the stable version.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3819#issuecomment-1535900177

**Reply by grahamIT** [reply]
Could you do a beta release that has a feature freeze? Having a feature freeze on a new branch would allow for bugs to be eliminated before a stable release. A feature freeze would also prevent new bugs from being introduced though new features.

**Comment by adamency** [comment]
Thanks for acknowledging this thread, I'll try reading the issue when I have the time.
